package.json

```diff
@@ -35,7 +35,7 @@
     },
     "dependencies": {
         "@orbs-network/ton-access": "^2.3.3",
-        "@tact-lang/compiler": "^1.3.0",
+        "@tact-lang/compiler": "^1.4.0",
```

Comment:
This change does not really matter, since `^` tells the package manager to upgrade Tact as long as it's version 1. So it will pick up the most recent 1.x version automatically

Reply:
True enough, my bad. We could close this PR in any case, since it has already been updated in main, but there is still this issue that tact 1.40 introduced a regression ? 

Reply:
The fix has been released with the `next` version of Tact, so you can upgrade like described in the Tact wiki: https://github.com/tact-lang/tact/wiki/Using-unreleased-versions-of-tools-early#dev-versions-of-tact-with-blueprint

Reply:
Another option would be to wait a bit for the upcoming Tact v1.4.1 release